app/extend/context.js

```diff
@@ -111,7 +112,12 @@ module.exports = {
     //  synchronize form requests' token always changing to protect against BREACH attacks
     if (token !== this[CSRF_SECRET] && !tokens.verify(this[CSRF_SECRET], token)) {
       debug('verify secret and token error');
-      this.throw(403, 'invalid csrf token');
+      this.throw(403, `Invalid csrf token. ${this[CSRF_NOTICE_SUFFIX]}`);
     }
   },
+
+  get [CSRF_NOTICE_SUFFIX]() {
+    if (this.app.config.env === 'local') return 'See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范';
```

Comment:
just in local env, it's ok I think. This also results in logs.

We can pending this pr util this part of english doc is translated. ok?